src/backend/app/auth/auth_routes.py

```diff
@@ -81,28 +91,21 @@ async def callback(request: Request, osm_auth=Depends(init_osm_auth)):
     # Get access token
     access_token = osm_auth.callback(callback_url).get("access_token")
     log.debug(f"Access token returned of length {len(access_token)}")
-    response = Response(status_code=HTTPStatus.OK)
 
-    # Set cookie
-    cookie_name = settings.FMTM_DOMAIN.replace(".", "_")
-    log.debug(
-        f"Setting cookie in response named '{cookie_name}' with params: "
-        f"max_age=31536000 | expires=31536000 | path='/' | "
-        f"domain={settings.FMTM_DOMAIN} | httponly=True | samesite='lax' | "
-        f"secure={False if settings.DEBUG else True}"
-    )
-    response.set_cookie(
-        key=cookie_name,
-        value=access_token,
-        max_age=31536000,  # OSM currently has no expiry
-        expires=31536000,  # OSM currently has no expiry
-        path="/",
-        domain=settings.FMTM_DOMAIN,
-        secure=False if settings.DEBUG else True,
-        httponly=True,
-        samesite="lax",
-    )
-    return response
+    osm_user = osm_auth.deserialize_access_token(access_token)
+    user_data = {
+        "id": osm_user["id"],
+        "sub": f"fmtm|{osm_user['id']}",
+        "aud": settings.FMTM_DOMAIN,
+        "iat": int(time.time()),
+        "exp": int(time.time()) + 86400,  # expiry set to 1 day
+        "username": osm_user["username"],
+        "email": osm_user.get("email"),
+        "img_url": osm_user.get("img_url"),
```

Comment:
I think the standard oauth claim for this is `picture` - worth double checking!

Reply:
its img_url,
```json
{'id': 18427944, 'username': 'Sujan Adhikari', 'img_url': 'https://www.openstreetmap.org/rails/active_storage/representations/redirect/eyJfcmFpbHMiOnsiZGF0YSI6MzI4NTE3MTksInB1ciI6ImJsb2JfaWQifX0=--6b73d47fac28854f227f3d0ecd56f2f6554c8709/eyJfcmFpbHMiOnsiZGF0YSI6eyJmb3JtYXQiOiJwbmciLCJyZXNpemVfdG9fbGltaXQiOlsxMDAsMTAwXX0sInB1ciI6InZhcmlhdGlvbiJ9fQ==--473f0190a9e741bb16565db85fe650d7b0a9ee69/sujan.webp'}
```

Reply:
https://www.iana.org/assignments/jwt/jwt.xhtml

The standard claim in the JWT spec is `picture`

`img_url` would be a custom claim